src/lib/Annotation.js

```diff
@@ -1,3 +1,4 @@
+import { flatten } from 'lodash';
```

Comment:
So importing this way will just reimport the entire lodash library... /me shakes my head at JavaScript... 

a work around is this: https://github.com/ProjectMirador/mirador/blob/edd913a3a05a79bbedf0237c9917e6818943cc77/src/lib/AnnotationResource.js#L2